sdk/compute/mgmt-v2020_10_01_preview/src/main/java/com/microsoft/azure/management/compute/v2020_10_01_preview/VirtualMachineRunCommands.java

```diff
@@ -40,7 +42,7 @@
      * @throws IllegalArgumentException thrown if parameters fail the validation
      * @return the observable for the request
      */
-    Observable<LocationVirtualMachineRunCommand> listAsync(final String location);
+    Observable<RunCommandDocumentBaseInner> listAsync(final String location);
```

Comment:
Quick fix for exposing the inner class instead of adding interface manually in lite SDK.